### DISCUSSION TITLE
Releases are mis-labeled

### DISCUSSION THREAD

**Discussion by PatrickF1** [DevToys-app/DevToys#1253] [General]
The homepage for the repo says the latest release is v1 from 1 year ago. It appears it is because all the v2 releases are labeled as "Pre-release". I would fix that labeling so it's more obvious that this project is active and recently updated.

<img width="193" alt="Screenshot 2024-06-20 at 9 15 26 AM" src="https://github.com/DevToys-app/DevToys/assets/1967248/160f41fd-4fbf-448e-8594-b7ab1992cc33">

**Comment by veler** [comment]
Unfortunately, in GitHub release, we can either select "pre-release" or "latest release" but not both. I think it's because GitHub consider "latest" as stable release. DevToys 2.0 isn't stable enough in my opinion so I'd like to keep it "pre-release" for now.

![image](https://github.com/DevToys-app/DevToys/assets/3747805/7f64ac6f-00b7-4412-83f4-d620253b2e30)

**Reply by PatrickF1** [reply]
Oh thanks. Sorry I didn't know it was intentional, esp. since Homebrew pulls v2.
